src/common/plugin/pluginState.ts

```diff
@@ -16,8 +16,8 @@ const SERVER_CACHE_FILE = 'serverCache.json'
 
 // Normalizes a URL to always be a websocket URL
 const asWebsocketUrl = (raw: unknown): string => {
-  const url = new URL(asString(raw))
-  return `wss://${url.host}/websocket`
+  const url = asString(raw)
+  return `${url}/websocket`
```

Comment:
Because of this change, all the info files need to be audit and the protocols need to be changed. The previous implementation "cleaned" every URL into the same shape, so it wouldn't matter if there was a `/websocket` suffix or not. Sense we're being stricter on the type, we need to regression test on all the currencies to make sure they sync.

Reply:
I did audit them and you'll notice I changed the ones that were wrong.